src/main/java/gregtech/api/unification/material/materials/ElementMaterials.java

```diff
@@ -0,0 +1,966 @@
+                .color(0x80C8F0)
+                .flags(EXT2_METAL, GENERATE_SMALL_GEAR, GENERATE_RING, GENERATE_FRAME)
+                .element(Elements.Al)
+                .toolStats(10.0f, 2.0f, 128)
+                .cableProperties(GTValues.V[4], 1, 1)
+                .fluidPipeProperties(1166, 35, true)
+                .blastTemp(1700)
+                .build();
+
+        Americium = new Material.Builder(3, "americium")
+                .ingot(3).fluid()
+                .color(0xC8C8C8).iconSet(METALLIC)
+                .flags(STD_METAL, GENERATE_ROD, GENERATE_LONG_ROD)
+                .element(Elements.Am)
+                .itemPipeProperties(64, 64)
+                .build();
+
+        Antimony = new Material.Builder(4, "antimony")
+                .ingot().fluid().ore()
+                .color(0xDCDCF0).iconSet(SHINY)
+                .flags(EXT_METAL, MORTAR_GRINDABLE)
+                .element(Elements.Sb)
+                .build();
+
+        Argon = new Material.Builder(5, "argon")
+                .fluid(FluidType.GAS).plasma()
+                .color(0x01FF01).iconSet(FLUID)
+                .element(Elements.Ar)
+                .build();
+
+        Arsenic = new Material.Builder(6, "arsenic")
+                .dust()
+                .element(Elements.As)
+                .build();
+
+        Astatine = new Material.Builder(7, "astatine")
+                .ingot().fluid()
+                .color(0x241A24)
+                .element(Elements.At)
+                .build();
+
+        Barium = new Material.Builder(8, "barium")
+                .ingot().fluid()
+                .iconSet(METALLIC)
+                .element(Elements.Ba)
+                .build();
+
+        Berkelium = new Material.Builder(9, "berkelium")
+                .ingot(3).fluid()
+                .color(0x645A88).iconSet(METALLIC)
+                .flags(EXT2_METAL)
+                .element(Elements.Bk)
+                .build();
+
+        Beryllium = new Material.Builder(10, "beryllium")
+                .ingot().fluid().ore()
+                .color(0x64B464).iconSet(METALLIC)
+                .flags(STD_METAL)
+                .element(Elements.Be)
+                .build();
+
+        Bismuth = new Material.Builder(11, "bismuth")
+                .ingot(1).fluid().ore()
+                .color(0x64A0A0).iconSet(METALLIC)
+                .element(Elements.Bi)
+                .build();
+
+        Bohrium = new Material.Builder(12, "bohrium")
+                .ingot(7).fluid()
+                .color(0xDC57FF).iconSet(SHINY)
+                .element(Elements.Bh)
+                .build();
+
+        Boron = new Material.Builder(13, "boron")
+                .dust()
+                .color(0xD2FAD2)
+                .element(Elements.B)
+                .build();
+
+        Bromine = new Material.Builder(14, "bromine")
+                .fluid()
+                .color(0x500A0A).iconSet(SHINY)
```

Comment:
Fluid iconsets are very strange right now. Currently (and before this PR), fluids could use one of three iconsets. The first was the regular Fluid one, which looks like water's texture. The second is the Gas iconset, which looks like a flat solid color, an example of this can be nitrogen dioxide. The last was Plasma, which is almost always the same looking as Gas due to how the texture looks. Any other iconset does not get applied to fluids. The only way to have a different fluid iconset right now is to specify Gas as the iconset for the material, or have a plasma variant. I have some ideas for a fluid iconset upgrade of sorts for a different PR in the future.

In short, the Shiny iconset (or any other, excluding Fluid, Gas, and Plasma) doesn't do anything to fluids right now, and in this case it will use the regular Fluid iconset and texture instead for the fluid form of the material.